src/Microsoft.Health.SqlServer/Features/Client/SqlConnectionWrapperFactory.cs

```diff
@@ -29,10 +30,10 @@ public class SqlConnectionWrapperFactory
             _sqlConnectionFactory = sqlConnectionFactory;
         }
 
-        public async Task<SqlConnectionWrapper> ObtainSqlConnectionWrapperAsync(bool enlistInTransaction = false)
+        public async Task<SqlConnectionWrapper> ObtainSqlConnectionWrapperAsync(CancellationToken cancellationToken, bool enlistInTransaction = false)
```

Comment:
Let's make this `(bool enlistInTransaction = false, CancellationToken cancellationToken = default)`.

Reply:
It was like that before but Smitha suggested to change it as CancellationToken is not usually defaulted in implementation. Should I default it to default and move it to the last?